### DISCUSSION TITLE
增加顶栏与顶栏下拉菜单的间隔

### DISCUSSION THREAD

**Discussion by kiyoakii** [the1812/Bilibili-Evolved#1764] [综合讨论 / General]
机型：Macbook Pro 16' 2019
Chrome 90.0.4430.85

当前顶栏和下拉菜单的间隔如图：

![image](https://user-images.githubusercontent.com/34620384/115657582-b6480980-a369-11eb-8278-073137ce875f.png)

感觉还挺不美观的，能否将菜单下移一点呢？

BTW，这个项目做得太漂亮了，也是真的好用，现在感觉自己的 BiliBili 清爽无比。

**Comment by the1812** [comment]
可以在控制台(Ctrl + Shift + I)里运行这个:
调整间距的大小可以修改里面的3个`4px`的数值
```js
bilibiliEvolved.installStyle({ name: 'navbarPopupMargin', displayName: '顶栏弹窗间距', style: `
.custom-navbar .popup {
  top: calc(100% + 4px) !important;
}
.custom-navbar .popup::before {
  content: "";
  position: absolute;
  top: -4px;
  left: 0;
  height: 4px;
  width: 100%;
  background: transparent;
}
`})
```
